crates/diesel_models/src/payment_attempt.rs

```diff
@@ -81,11 +62,19 @@ pub struct PaymentAttempt {
     pub profile_id: id_type::ProfileId,
     pub organization_id: id_type::OrganizationId,
     pub card_network: Option<String>,
+    pub payment_method_type_v2: Option<storage_enums::PaymentMethod>,
+    pub connector_payment_id: Option<String>,
```

Comment:
What is difference bewteen connector_payment_id and connector_response_reference_id?